src/api/app/views/webui/kiwi/images/_preferences.html.haml

```diff
@@ -0,0 +1,35 @@
+#kiwi-preferences
+  .nested-fields
+    %h2#image-name
+      Preferences
+    %ul.inline
+      %li
+        %strong Container config name:
+        %span.fill{ data: { tag: 'containerconfig_name' }}= @image.preference_type.containerconfig_name
```

Comment:
Maybe we can also add a description here what the container config name and tag is.